bazel/cargo/wasmtime/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 edition = "2021"
 name = "wasmtime-c-api-bazel"
-version = "9.0.3"
-rust-version = "1.66.0"
+version = "24.0.0"
+rust-version = "1.77.2"
```

Comment:
Wasmtime seems to require Rust v1.78.0 as MRV (see: https://github.com/bytecodealliance/wasmtime/blob/v24.0.0/Cargo.toml#L164).